app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,6 @@
+/- if signed_in?
```

Comment:
I think we shouldn't be committing commented code. It's in the git history if we need to reference it, and adds unnecessary noise when trying to read.  Same for line 6.

@teknofire Thoughts?